debian/rules

```diff
@@ -22,4 +22,6 @@ override_dh_install:
 	mkdir -p debian/tmp/var/lib/delphix-virtualization
 	docker pull registry.delphix.com/python:2.7.18-slim
 	docker save -o debian/tmp/var/lib/delphix-virtualization/docker-python-2-7-18-slim.tar registry.delphix.com/python:2.7.18-slim
+	docker pull registry.delphix.com/python:3.8-slim
```

Comment:
out of curiosity, does this pull from an external docker registry? or some how, pull from an artifactory mirror, or something we manage?

I know this isn't really a comment on this PR, but figured I'd ask in case you know the answer.

Reply:
From what I understand, this pulls from our mirror of [dockerhub](https://hub.docker.com/)